builds/misc/templates/build-packages.yaml

```diff
@@ -149,16 +149,31 @@ stages:
       ################################################################################
         displayName: Mariner_Linux
         pool:
-          vmImage: 'ubuntu-18.04'
+          # We are using Linux image that lives in Windows pool. The Windows pool by default uses Standard_D4s_v3 SKU.
+          # EFLOW requires ~85GB to build the edgelet artifacts.
+          name: $(pool.windows.name)
+          demands:
+          - ImageOverride -equals agent-aziotedge-ubuntu-20.04-docker-large-disk
         variables:
           NugetSecurityAnalysisWarningLevel: warn
         strategy:
           matrix:
             CBL-Mariner1.0-amd64:
               arch: amd64
               os: mariner
+              os_version: 1
               mariner_release: 1.0-stable
-              target.iotedged: builds/mariner/out/RPMS
+              target.iotedged: builds/mariner1/out/RPMS
+              target.logs: builds/mariner1/build/logs/pkggen/rpmbuilding/
+              IISCommit: 158a726f0684cdfa5609191c2c9ace0123530522
+            CBL-Mariner2.0-amd64:
+              arch: amd64
+              os: mariner
+              os_version: 2
+              mariner_release: 2.0-stable
+              target.iotedged: builds/mariner2/out/RPMS
+              target.logs: builds/mariner2/build/logs/pkggen/rpmbuilding/
+              IISCommit: 158a726f0684cdfa5609191c2c9ace0123530522
```

Comment:
1. Is there any particular reason why we pin the IISCommit to this particular commit? Don't we ideally want this value to be automatically parsed from the `cargo.lock` file? 
2. `IISCommit` variable should be job's variables (under `variables:`) section since it is used across two tasks.

Reply:
we have to use this commit since IIS has not had an official build after the Mariner 2.0 changes were added. After the next IIS release, we will be able to remove the IISCommit variable and pull it directly from cargo.lock